apps/glusterfs/listing.go

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright (c) 2018 The heketi Authors
+//
+// This file is licensed to you under your choice of the GNU Lesser
+// General Public License, version 3 or any later version (LGPLv3 or
+// later), or the GNU General Public License, version 2 (GPLv2), in all
+// cases as published by the Free Software Foundation.
+//
+
+package glusterfs
+
+import (
+	"github.com/boltdb/bolt"
+
+	"github.com/heketi/heketi/pkg/glusterfs/api"
+)
+
+// ListCompleteVolumes returns a list of volume ID strings for volumes
+// that are not pending.
+func ListCompleteVolumes(tx *bolt.Tx) ([]string, error) {
+	return VolumeList(tx)
+}
+
+// ListCompleteBlockVolumes returns a list of block volume ID strings for bricks
+// that are not pending.
+func ListCompleteBlockVolumes(tx *bolt.Tx) ([]string, error) {
+	return BlockVolumeList(tx)
+}
+
+// CompleteClusterInfoComplete updates the given ClusterInfoResponse object so
```

Comment:
I guess the comment needs to be updated.

Reply:
Yeah, that just got messed up. That's neither the original name nor the current name. :-)

Reply:
UpdateClusterInfoComplete it should be